### ISSUE TITLE
Document and sim test recruitment dev_vector options

### ISSUE THREAD

**Issue by k-doering-NOAA** [nmfs-stock-synthesis/stock-synthesis#29]
Imported from redmine, Issue [#49212](https://vlab.ncep.noaa.gov/redmine/issues/49212)
 Opened by @RickMethot on 2018-05-11
Status when imported: In Progress
 
Find while working with Cole Monnahan that the dev_vector approach to the recruitment deviations does not perform well in mceval_phase().

Three alternatives:
1.  ADMB developers find a fix;
2.  deprecate use of the dev_vector and add a penalty to the mean deviation to keep it close to 0.0.  Feasible because this mean covaries strongly with ln(R0).
3.  add the mean deviation to ln(R0) before using ln(R0) in the spawner-recruitment relationship; need to be careful about effects on early and forecast recruitments and on recruitments outside of the dev range.

**Comment by Rick-Methot-NOAA** [comment]
closely related to #37

**Comment by Rick-Methot-NOAA** [comment]
This issue really could use a simulation study to demonstrate the performance of the various options under MLE and MCMC.
@kellijohnson-NOAA 
@k-doering-NOAA

**Comment by k-doering-NOAA** [comment]
related to #37

**Comment by Rick-Methot-NOAA** [comment]
Isn't there another dec_dev issue that shows this being implemented, but needing performance testing; e.g. a paper.?

**Comment by k-doering-NOAA** [comment]
@Rick-Methot-NOAA , yes, we closed the other issue (#37) because it seemed to have more text about implementation of the option and we thought the performance testing was captured better by this open issue.
